hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileDeleteHandler.java

```diff
@@ -101,23 +105,22 @@ public void handle(
             throw new HandleException(FILE_DELETED);
         }
 
-        /* Copy all the fields from existing topic and change deleted flag */
-        final var fileBuilder = new File.Builder()
-                .fileNumber(file.fileNumber())
-                .expirationTime(file.expirationTime())
-                .keys(file.keys())
-                .contents(file.contents())
-                .memo(file.memo())
-                .deleted(true);
+        /* Copy all the fields from existing file and change deleted flag */
```

Comment:
Slightly misleading.  This code also removes the file contents (sets the contents to null).

Reply:
Still not accurate, as it does not copy all fields, it deletes the contents (irretrievably, in fact).

Reply:
fixed it.